src/vulkan/device.cc

```diff
@@ -390,16 +393,21 @@ Result Device::Initialize(
     PFN_vkGetInstanceProcAddr getInstanceProcAddr,
     Delegate* delegate,
     const std::vector<std::string>& required_features,
-    const std::vector<std::string>& required_extensions,
+    const std::vector<std::string>& required_instance_extensions,
+    const std::vector<std::string>& required_device_extensions,
     const VkPhysicalDeviceFeatures& available_features,
     const VkPhysicalDeviceFeatures2KHR& available_features2,
     const std::vector<std::string>& available_extensions) {
   Result r = LoadVulkanPointers(getInstanceProcAddr, delegate);
   if (!r.IsSuccess())
     return r;
 
-  bool use_physical_device_features_2 = false;
-  for (auto& ext : required_extensions) {
+  const bool isVulkan11Plus =
```

Comment:
Please match local naming conventions `is_vulkan_1_1_or_higher`

Reply:
I removed this change completely.